bin/gltf-pipeline.js

```diff
@@ -37,17 +29,10 @@ var fileExtension = path.extname(gltfPath);
 var fileName = path.basename(gltfPath, fileExtension);
 var filePath = path.dirname(gltfPath);
 
-<<<<<<< HEAD
 var outputPath = defaultValue(argv._[1], defaultValue(argv.o, argv.output));
 var binary = defaultValue(argv.b, defaultValue(argv.binary, false));
 var separate = defaultValue(argv.s, defaultValue(argv.separate, false));
 var separateImage = defaultValue(argv.t, defaultValue(argv.separateImage, false));
-=======
-var outputPath = defaultValue(argv._[1], argv.o);
-var isSeparate = defaultValue(argv.s, false);
-var isBinary = defaultValue(argv.b, false);
-var isQuantized = defaultValue(argv.q, false);
```

Comment:
All the quantized related code needs to stay in, but just change the naming like the others.